client/js/app/stores/ExplorerStore.js

```diff
@@ -283,6 +283,10 @@ function _wrapGroupBy(group_by) {
   return group_by;
 }
 
+function _coerceToNumber(str) {
+  return str && typeof(str) === 'string' ? Number(str) : str;
```

Comment:
You can just return `Number(str)` here, as `Number({a string value})` is the same as `Number({a number value})`.